data/colors/light.scss

```diff
@@ -404,6 +405,7 @@ $export: (
     secondary: $gray-0,
     tertiary: $gray-1,
     overlay: $white,
+    backdrop: $black-fade-15,
```

Comment:
Should this be `$black-fade-50`? Using `$black-fade-15` would be almost transparent:

`$black-fade-50` | `$black-fade-15`
--- | ---
![image](https://user-images.githubusercontent.com/378023/97649791-8e9f7500-1a9b-11eb-8fa8-9aaf15c13785.png) | ![Screen Shot 2020-10-30 at 10 32 43](https://user-images.githubusercontent.com/378023/97649755-7c253b80-1a9b-11eb-9396-de08344030f0.png)
 

```suggestion
    backdrop: $black-fade-50,
```